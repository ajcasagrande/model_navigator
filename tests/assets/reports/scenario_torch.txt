NavigatorEvent.WORKSPACE_INITIALIZED {'path': PosixPath('torch_workspace')}
NavigatorEvent.OPTIMIZATION_STARTED {}
NavigatorEvent.PIPELINE_STARTED {'name': 'Preprocessing'}
NavigatorEvent.COMMAND_STARTED {'command': 'InferInputMetadata', 'config_key': None, 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'FetchInputModelData', 'config_key': None, 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'InferOutputMetadata', 'config_key': None, 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'FetchOutputModelData', 'config_key': None, 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.PIPELINE_FINISHED {}
NavigatorEvent.PIPELINE_STARTED {'name': 'PyTorch Export'}
NavigatorEvent.COMMAND_STARTED {'command': 'ExportTorch2TorchScript', 'config_key': 'torchscript-script', 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'ExportTorch2TorchScript', 'config_key': 'torchscript-trace', 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'ExportExportedProgram', 'config_key': 'torch-exportedprogram', 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'ExportTorch2DynamoONNX', 'config_key': 'onnx-dynamo', 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.FAIL}
NavigatorEvent.COMMAND_STARTED {'command': 'GraphSurgeonOptimize', 'config_key': 'onnx-dynamo', 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
NavigatorEvent.COMMAND_STARTED {'command': 'ExportTorch2ONNX', 'config_key': 'onnx', 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'GraphSurgeonOptimize', 'config_key': 'onnx', 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.PIPELINE_FINISHED {}
NavigatorEvent.PIPELINE_STARTED {'name': 'Finding Max Batch Size for TensorRT'}
NavigatorEvent.COMMAND_STARTED {'command': 'FindMaxBatchSize', 'config_key': None, 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.PIPELINE_FINISHED {}
NavigatorEvent.PIPELINE_STARTED {'name': 'PyTorch Conversion'}
NavigatorEvent.COMMAND_STARTED {'command': 'ConvertTorchScript2ONNX', 'config_key': 'onnx-script', 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'GraphSurgeonOptimize', 'config_key': 'onnx-script', 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'ConvertTorchScript2ONNX', 'config_key': 'onnx-trace', 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'GraphSurgeonOptimize', 'config_key': 'onnx-trace', 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.PIPELINE_FINISHED {}
NavigatorEvent.PIPELINE_STARTED {'name': 'TensorRT Conversion'}
NavigatorEvent.COMMAND_STARTED {'command': 'ConvertONNX2TRT', 'config_key': 'trt-dynamo-fp32', 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
NavigatorEvent.COMMAND_STARTED {'command': 'ConvertONNX2TRT', 'config_key': 'trt-dynamo-fp16', 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
NavigatorEvent.COMMAND_STARTED {'command': 'ConvertONNX2TRT', 'config_key': 'trt-fp32', 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'ConvertONNX2TRT', 'config_key': 'trt-fp16', 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'ConvertONNX2TRT', 'config_key': 'trt-script-fp32', 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'ConvertONNX2TRT', 'config_key': 'trt-script-fp16', 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'ConvertONNX2TRT', 'config_key': 'trt-trace-fp32', 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'ConvertONNX2TRT', 'config_key': 'trt-trace-fp16', 'runner_name': None}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.PIPELINE_FINISHED {}
NavigatorEvent.PIPELINE_STARTED {'name': 'Correctness'}
NavigatorEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'torch', 'runner_name': 'TorchCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'torch', 'runner_name': 'TorchCompileCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'torchscript-script', 'runner_name': 'TorchScriptCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'torchscript-trace', 'runner_name': 'TorchScriptCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'torch-exportedprogram', 'runner_name': 'TorchExportedProgramCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.FAIL}
NavigatorEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'onnx-dynamo', 'runner_name': 'OnnxCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
NavigatorEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'onnx-dynamo', 'runner_name': 'OnnxTensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
NavigatorEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'onnx', 'runner_name': 'OnnxCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'onnx', 'runner_name': 'OnnxTensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'onnx-script', 'runner_name': 'OnnxCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'onnx-script', 'runner_name': 'OnnxTensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'onnx-trace', 'runner_name': 'OnnxCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'onnx-trace', 'runner_name': 'OnnxTensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'trt-dynamo-fp32', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
NavigatorEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'trt-dynamo-fp16', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
NavigatorEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'trt-fp32', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'trt-fp16', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'trt-script-fp32', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'trt-script-fp16', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'trt-trace-fp32', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Correctness', 'config_key': 'trt-trace-fp16', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.PIPELINE_FINISHED {}
NavigatorEvent.PIPELINE_STARTED {'name': 'Performance'}
NavigatorEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'torchscript-script', 'runner_name': 'TorchScriptCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'torchscript-trace', 'runner_name': 'TorchScriptCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'torch-exportedprogram', 'runner_name': 'TorchExportedProgramCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.FAIL}
NavigatorEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'onnx-dynamo', 'runner_name': 'OnnxCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
NavigatorEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'onnx-dynamo', 'runner_name': 'OnnxTensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
NavigatorEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'onnx', 'runner_name': 'OnnxCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'onnx', 'runner_name': 'OnnxTensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'onnx-script', 'runner_name': 'OnnxCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'onnx-script', 'runner_name': 'OnnxTensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'onnx-trace', 'runner_name': 'OnnxCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'onnx-trace', 'runner_name': 'OnnxTensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'trt-dynamo-fp32', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
NavigatorEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'trt-dynamo-fp16', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
NavigatorEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'trt-fp32', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'trt-fp16', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'trt-script-fp32', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'trt-script-fp16', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'trt-trace-fp32', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'trt-trace-fp16', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'torch', 'runner_name': 'TorchCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'Performance', 'config_key': 'torch', 'runner_name': 'TorchCompileCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.PIPELINE_FINISHED {}
NavigatorEvent.PIPELINE_STARTED {'name': 'Verify Models'}
NavigatorEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'torch', 'runner_name': 'TorchCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'torch', 'runner_name': 'TorchCompileCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'torchscript-script', 'runner_name': 'TorchScriptCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'torchscript-trace', 'runner_name': 'TorchScriptCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'torch-exportedprogram', 'runner_name': 'TorchExportedProgramCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
NavigatorEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'onnx-dynamo', 'runner_name': 'OnnxCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
NavigatorEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'onnx-dynamo', 'runner_name': 'OnnxTensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
NavigatorEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'onnx', 'runner_name': 'OnnxCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'onnx', 'runner_name': 'OnnxTensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'onnx-script', 'runner_name': 'OnnxCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'onnx-script', 'runner_name': 'OnnxTensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'onnx-trace', 'runner_name': 'OnnxCUDA'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'onnx-trace', 'runner_name': 'OnnxTensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'trt-dynamo-fp32', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
NavigatorEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'trt-dynamo-fp16', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.SKIPPED}
NavigatorEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'trt-fp32', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'trt-fp16', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'trt-script-fp32', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'trt-script-fp16', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'trt-trace-fp32', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.COMMAND_STARTED {'command': 'VerifyModel', 'config_key': 'trt-trace-fp16', 'runner_name': 'TensorRT'}
NavigatorEvent.COMMAND_FINISHED {'status': CommandStatus.OK}
NavigatorEvent.PIPELINE_FINISHED {}
NavigatorEvent.BEST_MODEL_PICKED {'config_key': 'torch', 'runner_name': 'TorchCUDA', 'model_path': None}
NavigatorEvent.OPTIMIZATION_FINISHED {}
