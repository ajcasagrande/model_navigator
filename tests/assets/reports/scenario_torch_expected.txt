Optimization started.
Model: Collecting model information ...
Model: Collecting model information OK
Model: Building torchscript-script model from Torch model ...
Model: Building torchscript-script model from Torch model OK
Model: Building torchscript-trace model from Torch model ...
Model: Building torchscript-trace model from Torch model OK
Model: Building ExportedProgram from Torch model ...
Model: Building ExportedProgram from Torch model OK
Model: Building ONNX Dynamo model from Torch model ...
Model: Building ONNX Dynamo model from Torch model FAIL
Model: Optimizing graph for onnx-dynamo ...
Model: Optimizing graph for onnx-dynamo SKIPPED
Model: Building ONNX Trace model from Torch model ...
Model: Building ONNX Trace model from Torch model OK
Model: Optimizing graph for onnx ...
Model: Optimizing graph for onnx OK
Model: Finding max batch size for TensorRT ...
Model: Finding max batch size for TensorRT OK
Model: Building ONNX model from TorchScript model ...
Model: Building ONNX model from TorchScript model OK
Model: Optimizing graph for onnx-script ...
Model: Optimizing graph for onnx-script OK
Model: Building ONNX model from TorchScript model ...
Model: Building ONNX model from TorchScript model OK
Model: Optimizing graph for onnx-trace ...
Model: Optimizing graph for onnx-trace OK
Model: Building TensorRT engine trt-dynamo-fp32 from ONNX model ...
Model: Building TensorRT engine trt-dynamo-fp32 from ONNX model SKIPPED
Model: Building TensorRT engine trt-dynamo-fp16 from ONNX model ...
Model: Building TensorRT engine trt-dynamo-fp16 from ONNX model SKIPPED
Model: Building TensorRT engine trt-fp32 from ONNX model ...
Model: Building TensorRT engine trt-fp32 from ONNX model OK
Model: Building TensorRT engine trt-fp16 from ONNX model ...
Model: Building TensorRT engine trt-fp16 from ONNX model OK
Model: Building TensorRT engine trt-script-fp32 from ONNX model ...
Model: Building TensorRT engine trt-script-fp32 from ONNX model OK
Model: Building TensorRT engine trt-script-fp16 from ONNX model ...
Model: Building TensorRT engine trt-script-fp16 from ONNX model OK
Model: Building TensorRT engine trt-trace-fp32 from ONNX model ...
Model: Building TensorRT engine trt-trace-fp32 from ONNX model OK
Model: Building TensorRT engine trt-trace-fp16 from ONNX model ...
Model: Building TensorRT engine trt-trace-fp16 from ONNX model OK
Model: Validating model torch on TorchCUDA backend ...
Model: Validating model torch on TorchCUDA backend OK
Model: Validating model torch on TorchCompileCUDA backend ...
Model: Validating model torch on TorchCompileCUDA backend OK
Model: Validating model torchscript-script on TorchScriptCUDA backend ...
Model: Validating model torchscript-script on TorchScriptCUDA backend OK
Model: Validating model torchscript-trace on TorchScriptCUDA backend ...
Model: Validating model torchscript-trace on TorchScriptCUDA backend OK
Model: Validating model torch-exportedprogram on TorchExportedProgramCUDA 
backend ...
Model: Validating model torch-exportedprogram on TorchExportedProgramCUDA 
backend FAIL
Model: Validating model onnx-dynamo on OnnxCUDA backend ...
Model: Validating model onnx-dynamo on OnnxCUDA backend SKIPPED
Model: Validating model onnx-dynamo on OnnxTensorRT backend ...
Model: Validating model onnx-dynamo on OnnxTensorRT backend SKIPPED
Model: Validating model onnx on OnnxCUDA backend ...
Model: Validating model onnx on OnnxCUDA backend OK
Model: Validating model onnx on OnnxTensorRT backend ...
Model: Validating model onnx on OnnxTensorRT backend OK
Model: Validating model onnx-script on OnnxCUDA backend ...
Model: Validating model onnx-script on OnnxCUDA backend OK
Model: Validating model onnx-script on OnnxTensorRT backend ...
Model: Validating model onnx-script on OnnxTensorRT backend OK
Model: Validating model onnx-trace on OnnxCUDA backend ...
Model: Validating model onnx-trace on OnnxCUDA backend OK
Model: Validating model onnx-trace on OnnxTensorRT backend ...
Model: Validating model onnx-trace on OnnxTensorRT backend OK
Model: Validating model trt-dynamo-fp32 on TensorRT backend ...
Model: Validating model trt-dynamo-fp32 on TensorRT backend SKIPPED
Model: Validating model trt-dynamo-fp16 on TensorRT backend ...
Model: Validating model trt-dynamo-fp16 on TensorRT backend SKIPPED
Model: Validating model trt-fp32 on TensorRT backend ...
Model: Validating model trt-fp32 on TensorRT backend OK
Model: Validating model trt-fp16 on TensorRT backend ...
Model: Validating model trt-fp16 on TensorRT backend OK
Model: Validating model trt-script-fp32 on TensorRT backend ...
Model: Validating model trt-script-fp32 on TensorRT backend OK
Model: Validating model trt-script-fp16 on TensorRT backend ...
Model: Validating model trt-script-fp16 on TensorRT backend OK
Model: Validating model trt-trace-fp32 on TensorRT backend ...
Model: Validating model trt-trace-fp32 on TensorRT backend OK
Model: Validating model trt-trace-fp16 on TensorRT backend ...
Model: Validating model trt-trace-fp16 on TensorRT backend OK
Model: Benchmarking model torchscript-script on TorchScriptCUDA backend ...
Model: Benchmarking model torchscript-script on TorchScriptCUDA backend OK
Model: Benchmarking model torchscript-trace on TorchScriptCUDA backend ...
Model: Benchmarking model torchscript-trace on TorchScriptCUDA backend OK
Model: Benchmarking model torch-exportedprogram on TorchExportedProgramCUDA 
backend ...
Model: Benchmarking model torch-exportedprogram on TorchExportedProgramCUDA 
backend FAIL
Model: Benchmarking model onnx-dynamo on OnnxCUDA backend ...
Model: Benchmarking model onnx-dynamo on OnnxCUDA backend SKIPPED
Model: Benchmarking model onnx-dynamo on OnnxTensorRT backend ...
Model: Benchmarking model onnx-dynamo on OnnxTensorRT backend SKIPPED
Model: Benchmarking model onnx on OnnxCUDA backend ...
Model: Benchmarking model onnx on OnnxCUDA backend OK
Model: Benchmarking model onnx on OnnxTensorRT backend ...
Model: Benchmarking model onnx on OnnxTensorRT backend OK
Model: Benchmarking model onnx-script on OnnxCUDA backend ...
Model: Benchmarking model onnx-script on OnnxCUDA backend OK
Model: Benchmarking model onnx-script on OnnxTensorRT backend ...
Model: Benchmarking model onnx-script on OnnxTensorRT backend OK
Model: Benchmarking model onnx-trace on OnnxCUDA backend ...
Model: Benchmarking model onnx-trace on OnnxCUDA backend OK
Model: Benchmarking model onnx-trace on OnnxTensorRT backend ...
Model: Benchmarking model onnx-trace on OnnxTensorRT backend OK
Model: Benchmarking model trt-dynamo-fp32 on TensorRT backend ...
Model: Benchmarking model trt-dynamo-fp32 on TensorRT backend SKIPPED
Model: Benchmarking model trt-dynamo-fp16 on TensorRT backend ...
Model: Benchmarking model trt-dynamo-fp16 on TensorRT backend SKIPPED
Model: Benchmarking model trt-fp32 on TensorRT backend ...
Model: Benchmarking model trt-fp32 on TensorRT backend OK
Model: Benchmarking model trt-fp16 on TensorRT backend ...
Model: Benchmarking model trt-fp16 on TensorRT backend OK
Model: Benchmarking model trt-script-fp32 on TensorRT backend ...
Model: Benchmarking model trt-script-fp32 on TensorRT backend OK
Model: Benchmarking model trt-script-fp16 on TensorRT backend ...
Model: Benchmarking model trt-script-fp16 on TensorRT backend OK
Model: Benchmarking model trt-trace-fp32 on TensorRT backend ...
Model: Benchmarking model trt-trace-fp32 on TensorRT backend OK
Model: Benchmarking model trt-trace-fp16 on TensorRT backend ...
Model: Benchmarking model trt-trace-fp16 on TensorRT backend OK
Model: Benchmarking model torch on TorchCUDA backend ...
Model: Benchmarking model torch on TorchCUDA backend OK
Model: Benchmarking model torch on TorchCompileCUDA backend ...
Model: Benchmarking model torch on TorchCompileCUDA backend OK
Model: Verifying model torch on TorchCUDA backend ...
Model: Verifying model torch on TorchCUDA backend OK
Model: Verifying model torch on TorchCompileCUDA backend ...
Model: Verifying model torch on TorchCompileCUDA backend OK
Model: Verifying model torchscript-script on TorchScriptCUDA backend ...
Model: Verifying model torchscript-script on TorchScriptCUDA backend OK
Model: Verifying model torchscript-trace on TorchScriptCUDA backend ...
Model: Verifying model torchscript-trace on TorchScriptCUDA backend OK
Model: Verifying model torch-exportedprogram on TorchExportedProgramCUDA backend
...
Model: Verifying model torch-exportedprogram on TorchExportedProgramCUDA backend
SKIPPED
Model: Verifying model onnx-dynamo on OnnxCUDA backend ...
Model: Verifying model onnx-dynamo on OnnxCUDA backend SKIPPED
Model: Verifying model onnx-dynamo on OnnxTensorRT backend ...
Model: Verifying model onnx-dynamo on OnnxTensorRT backend SKIPPED
Model: Verifying model onnx on OnnxCUDA backend ...
Model: Verifying model onnx on OnnxCUDA backend OK
Model: Verifying model onnx on OnnxTensorRT backend ...
Model: Verifying model onnx on OnnxTensorRT backend OK
Model: Verifying model onnx-script on OnnxCUDA backend ...
Model: Verifying model onnx-script on OnnxCUDA backend OK
Model: Verifying model onnx-script on OnnxTensorRT backend ...
Model: Verifying model onnx-script on OnnxTensorRT backend OK
Model: Verifying model onnx-trace on OnnxCUDA backend ...
Model: Verifying model onnx-trace on OnnxCUDA backend OK
Model: Verifying model onnx-trace on OnnxTensorRT backend ...
Model: Verifying model onnx-trace on OnnxTensorRT backend OK
Model: Verifying model trt-dynamo-fp32 on TensorRT backend ...
Model: Verifying model trt-dynamo-fp32 on TensorRT backend SKIPPED
Model: Verifying model trt-dynamo-fp16 on TensorRT backend ...
Model: Verifying model trt-dynamo-fp16 on TensorRT backend SKIPPED
Model: Verifying model trt-fp32 on TensorRT backend ...
Model: Verifying model trt-fp32 on TensorRT backend OK
Model: Verifying model trt-fp16 on TensorRT backend ...
Model: Verifying model trt-fp16 on TensorRT backend OK
Model: Verifying model trt-script-fp32 on TensorRT backend ...
Model: Verifying model trt-script-fp32 on TensorRT backend OK
Model: Verifying model trt-script-fp16 on TensorRT backend ...
Model: Verifying model trt-script-fp16 on TensorRT backend OK
Model: Verifying model trt-trace-fp32 on TensorRT backend ...
Model: Verifying model trt-trace-fp32 on TensorRT backend OK
Model: Verifying model trt-trace-fp16 on TensorRT backend ...
Model: Verifying model trt-trace-fp16 on TensorRT backend OK
Optimization finished for the model.
   Optimization result for max throughput and min latency   
                          strategy                          
┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃ Optimized backend          ┃ Path                        ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
│ torch on TorchCUDA backend │ NA (source model is picked) │
└────────────────────────────┴─────────────────────────────┘
